src/twisted/conch/scripts/ckeygen.py

```diff
@@ -150,7 +150,7 @@ def generateECDSAkey(options):
 
     if not options['bits']:
         options['bits'] = 256
-    # OpenSSH supports only nistp curves.
+    # OpenSSH supports only mandatory sections of RFC5656.
```

Comment:
but that is what it says in the release notes https://www.openssh.com/txt/release-5.7. 

> Only the mandatory sections of RFC5656 are implemented, specifically
   the three REQUIRED curves nistp256, nistp384 and nistp521 and only
   ECDH and ECDSA. Point compression (optional in RFC5656) is NOT
   implemented